### PR TITLE
Letters copy edits 08/17/2017

### PR DESCRIPTION
**Pull request by goldenmeanie**
PL copy updates added. Will be easier for @peggygannon to review on staging.

References [#6527](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3947#issuecomment-323076527)

### CODE REVIEW

**Comment by goldenmeanie** [comment]
@peggygannon you should be able to look at this PR to confirm edits now. But it might be easier to get it deployed to staging given the login requirement.

**Comment by rroueche** [comment]
@annekainicUSDS pinging to see if you could review and merge this quickly today, so peggy and I can get in and take a peek to make sure it's :100: prior to the stakeholder review on Tues.

File: src/js/letters/containers/DownloadLetters.jsx [hunk 1]
```diff
@@ -10,41 +10,41 @@ class DownloadLetters extends React.Component {
...
+          <li><a href="https://eauth.va.gov/ebenefits/coe" target="_blank"><strong>Request a Certificate of Eligibility (COE) for your home loan benefits.</strong></a></li>
+          <li><a href="https://eauth.va.gov/ebenefits/DPRIS" target="_blank"><strong>Request a copy of your discharge or separation papers (DD214).</strong></a></li>
         </ul>
-        <div className="feature help-desk">
-          <h2>Get Adobe Reader</h2>
-          <div>To download and view your letters, please make sure you have the latest free version of Adobe Reader.</div>
-          <a href="https://get.adobe.com/reader/" target="_blank">Get Adobe Reader</a>
-        </div>
+        <p>
+          Please visit eBenefits [link to eBenefits] for any document or letter not listed here.
```
Comment (annekainicUSDS) [inline src/js/letters/containers/DownloadLetters.jsx]: Did you want to include the actual link here?

**Review by annekainicUSDS — APPROVED** [review]
LGTM with one minor comment!

**Comment by annekainicUSDS** [comment]
There's a failing test here too. I can fix that and push up the change.

**Comment by rroueche** [comment]
@annekainicUSDS yes please! i didn't know if you were asking or not (sorry haha)

**Comment by annekainicUSDS** [comment]
@rroueche @goldenmeanie great, I will make the fixes to this PR!

**Comment by goldenmeanie** [comment]
Thanks for fixing that test @annekainicUSDS! I'd like to check in with you about that conditional bit before we merge!

**Comment by annekainicUSDS** [comment]
@goldenmeanie Ah yes, I didn't thoroughly review the logic there, but I think it's wrong. Let's talk in more detail about the change before merging.

**Comment by annekainicUSDS** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/vets-website/pull/6244

**Pull request closed without merging**